core/survey/_survey/surveyRefDataIndex.js

```diff
@@ -52,83 +52,82 @@ const keys = {
   taxonIndex: 'taxonIndex',
 }
 
+const categoryItemNullParentUuid = 'null'
```

Comment:
Why is this string `'null'` and not just `null`?

Reply:
because it's used as a key in the dictionary of category items by parent item uuid